java/jdbc/src/main/java/com/flipkart/vitess/jdbc/VitessVTGateManager.java

```diff
@@ -95,8 +112,8 @@ private static VTGateConn getVtGateConn(String hostname, int port, String userna
      * @param username
```

Comment:
@param keyspace availble in new changes.

 +     * Create vtGateConn object with given identifier.
+     *
+     * @param hostname
+     * @param port
+     * @param username
+     * @param keyspace
+     * @return
+     */